WcaOnRails/app/models/competition.rb

```diff
@@ -66,10 +65,10 @@ class Competition < ActiveRecord::Base
     self.persisted? || (name.length <= MAX_NAME_LENGTH && name =~ VALID_NAME_RE)
   end
   validates :name, length: { maximum: MAX_NAME_LENGTH },
-                   format: { with: VALID_NAME_RE, message: INVALID_NAME_MESSAGE }
+                   format: { with: VALID_NAME_RE, message: I18n.t('competitions.errors.invalid_name_message') }
```

Comment:
We got a chat offline, here is what @jfly is talking about:
![mixed-locale](https://cloud.githubusercontent.com/assets/1007485/18526465/fef9cb36-7ac0-11e6-84f4-ab0ca9b2cd96.png)

I first loaded the error in English, then switch locale when doing the post, and it doesn't seem to use the correct validation error.